firecrown/ccl/sources/sources.py

```diff
@@ -374,3 +374,80 @@ def render(self, cosmo, params, systematics=None):
             systematics[systematic].apply(cosmo, params, self)
 
         self.dndz_interp_ = Akima1DInterpolator(self.z_, self.dndz_)
+
+
+class CMBLSource(Source):
+    """A CCL CMB Lensing Source.
+
+    Parameters
+    ----------
+    sacc_tracer : str
+        The name of the tracer in the SACC file.
+    scale : float, optional
+        The default scale for this source. Usually the default of 1.0 is
+        correct.
+    systematics : list of str, optional
+        A list of the source-level systematics to apply to the source. The
+        default of `None` implies no systematics.
+
+    Attributes
+    ----------
+    beam_orig : np.ndarray
+        The original beam used in the sacc tracer before any systematics
+        are applied. Set after a call to `read`.
+    beam_: np.ndarray
+        The beam used for CMB lensing. Set after a call to `render`.
+    scale_ : float
+        The overall scale associated with the source. Set after a call to
+        `render`.
+    tracer_ : `pyccl.WeakLensingTracer`
```

Comment:
Is this right?

Reply:
Thanks, yes the tracer should be a cmb tracer.